examples/react/react-app/src/App.tsx

```diff
@@ -99,7 +113,15 @@ function App() {
             </div>
 
             <div className="card">
-                <button onClick={() => spawn(account.account)}>Spawn</button>
+                <button
+                    onClick={() =>
+                        dojo_starter.actions.spawn({
+                            account: account.account.address,
```

Comment:
It could be nice to support passing an account configuration object to the dojo client that we pass to `DojoStarter`, so we don't always have to specify this, and then supporting optionally overloading the account at the call site

Reply:
Yup, was exactly my same thought! Felt like there was a lot of repetition wrt to the address.

> 99% (100 even?) of the times, the calls will come from the logged in user and so would improve DX as well.